nevergrad/functions/images/imagelosses.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+from koncept.models import Koncept512
+
+
+class _ImageLoss:
+    def __init__(self, reference=None) -> None:
+        self.reference = reference
+
+    def __call__(self, img: np.ndarray) -> float:
+        pass
+
+
+class SumAbsoluteDifferencesLoss(_ImageLoss):
```

Comment:
Given the module and the class, it's clear they are losses. If we import the module we get `imagelosses.SumAbsoluteDifferencesLoss` which would be redundant long and not helping, so better remove it for all but the base class
```suggestion
class SumAbsoluteDifferences(ImageLoss):
```